### PR TITLE
Fix the LMDB backend not finding symbols in `pdns-auth` and `pdns-auth-util`

### PR DESCRIPTION
**Pull request by fredmorcos**
### Short description
Fix the LMDB backend not finding symbols in `pdns-auth` and `pdns-auth-util`. This ended up being much simpler than the original route I had taken of doing lots of refactoring. I went down the wrong path because I was under the impression that some of those missing symbols were missing in the module itself and not in the executable that is loading the module (e.g. `pdns-auth` or `pdns-auth-util`).

### Checklist
I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled this code
- [X] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)

### CODE REVIEW

File: modules/lmdbbackend/meson.build [hunk 1]
```diff
@@ -6,4 +6,4 @@ module_extras = files(
   'lmdbbackend.hh',
 )
 
-module_deps = [deps, dep_lmdb_safe, dep_lmdb, dep_boost_serialization]
+module_deps = [deps, dep_lmdb_safe, dep_lmdb, dep_boost_serialization, dep_systemd]
```
Comment (Habbie) [inline modules/lmdbbackend/meson.build:9]: tihs seems weird

Comment (fredmorcos) [reply modules/lmdbbackend/meson.build:9]: I thought so. When I was debugging the other issue and looked at the autotools `Makefile.am` for the LMDB backend, I found the following:
```
liblmdbbackend_la_LIBADD = $(LMDB_LIBS) $(BOOST_SERIALIZATION_LIBS) $(SYSTEMD_LIBS)
```
And that's why I added it here.

Comment (fredmorcos) [reply modules/lmdbbackend/meson.build:9]: It is, in fact, needed. From `lmdbbackend.cc`:
```
#ifdef HAVE_SYSTEMD
  /* A schema migration may take a long time. Extend the startup service timeout to 1 day,
   * but only if this is beyond the original maximum time of TimeoutStartSec=.
   */
  sd_notify(0, "EXTEND_TIMEOUT_USEC=86400000000");
#endif
```

Comment (Habbie) [reply modules/lmdbbackend/meson.build:9]: oh right!

**Review by Habbie — APPROVED** [review]

**Pull request merged**